README.md

```diff
@@ -19,10 +19,49 @@ Using GraphQL with server-side rendering in React is a challenging problem. Curr
 
 ### Installation
 
+Install `next-urql` along with its `peerDependencies`.
+
 ```sh
-yarn add next-urql
+yarn add next-urql react-is styled-components isomorphic-unfetch
```

Comment:
I personally like to add `npm i` here in case people aren't familiar with `yarn` but that's personal pref